c2rust/Cargo.toml

```diff
@@ -1,34 +1,34 @@
 [package]
-name = "c2rust"
-description = "C to Rust translation, refactoring, and cross-checking"
-version = "0.16.0"
 authors = ["The C2Rust Project Developers <c2rust@immunant.com>"]
-license = "BSD-3-Clause"
-homepage = "https://c2rust.com/"
-repository = "https://github.com/immunant/c2rust"
-edition = "2021"
 categories = ["development-tools", "development-tools::ffi", "command-line-utilities"]
+default-run = "c2rust"
+description = "C to Rust translation, refactoring, and cross-checking"
+edition = "2021"
+homepage = "https://c2rust.com/"
 keywords = ["transpiler", "migration", "translation", "c"]
+license = "BSD-3-Clause"
+name = "c2rust"
 readme = "README.md"
-default-run = "c2rust"
+repository = "https://github.com/immunant/c2rust"
+version = "0.16.0"
 
 [badges]
-travis-ci = { repository = "immunant/c2rust" }
-azure-devops = { project = "immunant/c2rust", pipeline = "immunant.c2rust", build="1" }
+azure-devops = {project = "immunant/c2rust", pipeline = "immunant.c2rust", build = "1"}
+travis-ci = {repository = "immunant/c2rust"}
```

Comment:
Please also don't reformat, especially because `{ key` is more idiomatic than `{key` in `Cargo.toml`s.